test/test_events.py

```diff
@@ -157,7 +157,7 @@ def method(self, arg):
             stack = "".join(traceback.format_exception(exc_type, exc, tb))
             assert "NotYet: try again later" in stack
             assert "nameko_amqp_retry.backoff.Backoff" in stack
-            assert "nameko_amqp_retry.backoff.Expired" in stack
+            assert "nameko_amqp_retry.backoff.Backoff.Expired" in stack
```

Comment:
This isn't a typo actually. The traceback is expected to contain `nameko_amqp_retry.backoff.Expired` despite the nested class declaration. I believe the string format is `<modulepath>.<classname>`.

Reply:
It turns out that this is dependent on the version of Python. On 3.4, the string format is `<modulepath>.<classname>`; on newer Pythons, the encoding class is included.